src/Http/Place/ImportPlaceRequestHandler.php

```diff
@@ -120,6 +123,11 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         $place = RequestBodyParserFactory::createBaseParser(
             new IdPropertyPolyfillRequestBodyParser($this->iriGenerator, $placeId),
             $this->importPreProcessingRequestBodyParser,
+            new JsonSchemaValidatingRequestBodyParser(JsonSchemaLocator::PLACE),
+            new CalendarValidationRequestBodyParser(),
+            new BookingInfoValidationRequestBodyParser(),
+            new PriceInfoValidationRequestBodyParser(),
+            MainLanguageValidatingRequestBodyParser::createForPlace(),
```

Comment:
We also need a validator that checks that the `eventtype` term that is used is usable on places. That is currently done here in the Respect validator: https://github.com/cultuurnet/udb3-silex/blob/e301148e32d3a8a0f061f931b50a1240ce670ab6/src/Model/Import/Validation/Place/PlaceImportValidator.php#L31

Reply:
The `ImportTermRequestBodyParser` now checks also if the `eventtype` is supported or not.

Reply:
I created it, but it lacks context 😅